transforms/lpc55s69_cm33_core0.yaml

```diff
@@ -0,0 +1,22 @@
+transforms:
+  - !DeleteEnumVariants
```

Comment:
A number of these are duplicated on core1. We could probably include a common patch for both.